sctp.c

```diff
@@ -259,12 +259,19 @@ janus_sctp_association *janus_sctp_association_create(void *dtls, uint64_t handl
 	GError *error = NULL;
 	char tname[16];
 	g_snprintf(tname, sizeof(tname), "sctp %"SCNu64, sctp->handle_id);
+	void *component = ((janus_dtls_srtp *)dtls)->component;
+	if (component) {
+		janus_refcount_increase(&((janus_ice_component*)component)->ref);
```

Comment:
Just a nit for code style: please separate the type and the asterisk with a space (in that other line too).

Reply:
Done